engine/src/flutter/lib/web_ui/lib/semantics.dart

```diff
@@ -265,6 +265,257 @@ class SemanticsFlag {
   String toString() => 'SemanticsFlag.$name';
 }
 
+/// Semantics flags.
```

Comment:
We don't include the dartdocs in the web copies of the `dart:ui` API (this way they don't get stale).

This comment applies here and below.